CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 ##  Unreleased
 
+##  [2.9.1] 2022-12-27
+
+### Changed/Added
+- Roo::Base#each_with_pagename returns Enumerator Object [576](https://github.com/roo-rb/roo/pull/576)
```

Comment:
Please do not make new version, write it in Unreleased paragraph because there is no release schedule not yet.